corehq/apps/app_manager/models.py

```diff
@@ -3258,7 +3258,7 @@ def create_profile(self, is_odk=False, with_media=False, template='app_manager/p
             'uniqueid': self.copy_of or self.id,
             'name': self.name,
             'descriptor': u"Profile File"
-        }).decode('utf-8')
+        }).encode('utf-8')
```

Comment:
i'm going to let @millerdev comment on this one. @emord are you thinking that this has been broken all along and the only reason we weren't seeing it is that the `decode` call was a noop?